articles/avoid-fix-flaky-tests.md

```diff
@@ -0,0 +1,73 @@
+---
+authorName: Adrian Herzog
+authorGithubUsername: adiherzog
+issue: 42
+title: How to avoid / deal with flaky system tests
+---
+# {{page.title}}
+
+There are many reasons to avoid automated tests that check an entire system or even a group of connected systems. But there are also good reasons to build and maintain a reasonable number of such tests. For this article, let's assume that the tests are there for a good reason and provide valuable information to the team.
+
+Now, what can you do if one or more of these tests are flaky? By flaky I mean that the test changes from green to red or vice versa without any changes to the code.
+
+Here are a number of strategies that hopefully will help you keeping the tests stable.
+
+
+# Increase Your Chances To Succeed
+
+## Only Write Automated Tests For What's Under Your Control
```

Comment:
I don't like that recommendation:
With which tests do I know if my 3rd party didn't break the contract?
With which tests do I know the overall system works? aka Continuos Health Tests in Production?

1. Make clear that you don't cover those, or 2. include them?
What about option 1?